website/common/locales/en/emptyArmory.json

```diff
@@ -0,0 +1,22 @@
+{
+    "emptyArmoryNew": "Empty the armory: New Challenge Available!",
+    "emptyArmoryUnlock": "You've unlocked 'Empty the Armory'! This special Market item allows you to reset your equipment. Use it to breathe new challenge into Habitica if you feel you've achieved it all, or if you just like running around naked!",
+    "emptyArmoryBegin": "Empty the armory: Begin a New Challenge",
+    "emptyArmoryStartOver": "Empty the armory resets your equipment to zero.",
+    "emptyArmoryAdvList1": "You return to zero equipment.",
+    "emptyArmoryAdvList2": "Except for zero gold cost equipment (gem items, contributor items, etc)",
+    "emptyArmoryEarnAchievement": "You also earn an Achievement for beginning a new adventure!",
+    "beEmptyArmory": "Empty the armory",
+    "emptyArmoryAchievement": "You've begun a new challenge! This is empty the armory <%= number %> for you, and the highest Level you've attained is <%= level %>. To stack this Achievement, begin your next new challenge when you've reached an even higher Level!",
+    "emptyArmoryAchievement100": "You've begun a new challenge! This is empty the armory <%= number %> for you, and the highest Level you've attained is 100 or higher. To stack this Achievement, begin your next new challenge when you've reached at least 100!",
+    "emptyArmoryBegan": "Began a New Challenge",
+    "emptyArmoryText": "Began <%= emptyArmorys %> New Challenges",
+    "emptyArmoryKey": "Used an armory key to reset equipment after attaining Level <%= level %>.",
+    "emptyArmoryKey100": "Used an armory key to start over after attaining Level 100 or higher.",
+    "emptyArmoryKeyNoLevel": "Used an armory key to start over.",
+    "emptyArmoryPop": "Instantly reset your equipment!",
+    "emptyArmoryName": "Armory key",
+    "reset": "Reset Equipment, max level <%= reLevel %>",
+    "confirmReset": "Are you sure?",
+    "emptyArmoryComplete": "You have reset your equipment!"
+}
```

Comment:
I think not all of the strings above are used in your new code. You can delete any that aren't used. 

I'm guessing the unused ones are because they've been copied from the rebirth.json file. There's probably a bunch of strings in that file that aren't used anywhere. That's because changes have been over time without a full clean-up of the unused strings. We don't need to exactly duplicate what's been done for rebirth in cases like that.

It's probably unnecessary to have a dedicated locale file for this (although I realise rebirth has one). There's extra work needed related to translations when a new locale file is created, so avoiding that is best when it's not essential. My personal feeling is that the new strings could go into the achievements.json file but if any of the staff say otherwise, follow their advice not mine.

Once any unnecessary new strings have been removed, we can work on changing the text if needed.

Reply:
Actually you are quite right... That was the result of a cheap copy/paste at the beginning, and I forgot to look more back into it at the end. It seems nearly half of those strings are not being used actually, so yeah might be better to put it in achievements.json file. Thanks :)

Reply:
I'm wondering if the word "challenge" will be confusing to some players, although I can't think of other phrasing that could be used. Rebirth uses "adventure" but I think you're right to not copy that here because it would be nice to have different phrasing between the two (and buying gear is less adventurous than starting over from scratch!)

Apart from that I have no further comments about these strings. Thanks for the changes you've made!

I haven't reviewed the code or done any testing.

Reply:
New... closet? Trunk? Armory? Load of clothes? Fashion line?

I don't know how helpful this is but I wanted to try. XP

(The other alternative I have in my head is to go whole hog and phrase it like the user *lost* something, like "Oh dear! You've broken the lock to your [equipment place name]! Nothing to do now but take this fresh key and start a new one! This is [#] for you, and you've reached [level]." or "Someone's broken into your equipment stash and taken it all!" or something. XD)